docs/src/main/paradox/project/scala3.md

```diff
@@ -1,15 +1,9 @@
 # Scala 3 support
 
-Pekko has experimental support for Scala 3.
+Pekko has support for Scala 3. Now that the Scala 3.3 LTS release is out, we can be pretty confident that it is unlikely that we will need to make binary incompatible changes due to Scala 3 changes. See the @ref:[binary compatibility documentation](../common/binary-compatibility-rules.md).
```

Comment:
`that it is unlikely that` can be optimized for reading

Reply:
`Now with Scala 3.3 LTS we can be confident no binary incompatible changes will be necessary due to Scala 3 changes`